platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerViewModel.swift

```diff
@@ -656,3 +750,18 @@ extension WysiwygComposerViewModel: ComposerModelWrapperDelegate {
 private extension Logger {
     static let viewModel = Logger(subsystem: subsystem, category: "ViewModel")
 }
+
+private struct ReplaceTextUpdate {
+    static let debounceThreshold = 0.1
+    var date: Date
+    var range: NSRange
+    var text: String
+}
+
+private extension ReplaceTextUpdate {
+    func isDuplicate(of other: ReplaceTextUpdate) -> Bool {
+        range == other.range
+            && text == other.text
+            && fabs(date.timeIntervalSince(other.date)) < Self.debounceThreshold
```

Comment:
why is this necessary?

Reply:
There is a descriptive comment where this function is called. We get 2 text updates with the same values when we tap on predictive inline text. This function is used to determine if we should ignore the second of these updates. `isDuplicate` is a bit misleading. `shouldDebounce` might be clearer.